src/util/tutorial.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import type { EdgeAccount } from 'edge-core-js'
+
+const TUTORIAL = 'tutorial.json'
+
+export const getUserLoginCount = async (account: EdgeAccount): Promise<number> => {
+  const { disklet, id } = account
+  try {
+    const userData = JSON.parse(await disklet.getText(TUTORIAL))
+    return userData[id] ? parseInt(userData[id].loginCount) : 0
```

Comment:
there is no need to index by account id. the disklet on the account is already account specific

Reply:
fixed